ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+
+workflow:
+  as: openshift-qe-node-density-cni
+  steps:
+    test:
+      - ref: openshift-qe-node-density-cni
+      - ref: openshift-qe-write-perfscale-results
+  env:
```

Comment:
Based on some of the errors I think this **env** at the wrong tabbing in the file, might need to take a look